app/experimenter/nimbus-ui/templates/nimbus/404.html

```diff
@@ -0,0 +1,164 @@
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="icon" href="{% static "imgs/favicon.ico" %}" />
+    <meta name="description" content="" />
+    <meta name="referrer" content="origin" />
+    <meta name="robots" content="noindex,nofollow" />
+    <meta
+      name="viewport"
+      content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=2,user-scalable=yes"
+    />
+    <title>Mozilla Experimenter</title>
+
+    <link rel="stylesheet" href="{% static 'nimbus/static/css/404.css' %}" />
+  </head>
+
+  <body>
+    <div class="background">
+      <div class="sea">
+        <div class="not-found">
+          <p>
+            Page not found.
+          </p>
+          <span>
+            Your flying nimbus must be mistaken —
+            <a href="/">go home</a>?
```

Comment:
This does go to the legacy UI because I have this 404 page showing for _all_ 404s (Nimbus or non-Nimbus), but maybe it makes sense to go to `/nimbus/`. I imagine eventually we'll retire `/nimbus/` in favor of showing the Nimbus console at `/`, but the legacy UI is probably seeing less use now.

Reply:
Two links? One for “Legacy” and one for /nimbus/?